Source/Details/ASTraitCollection.m

```diff
@@ -32,6 +32,14 @@ ASDISPLAYNODE_INLINE UIContentSizeCategory AS_UIContentSizeCategoryUnspecified()
   }
 }
 
+ASDISPLAYNODE_INLINE ASPrimitiveContentSizeCategory safePrimitiveContentSizeCategory(ASPrimitiveContentSizeCategory sizeCategory) {
+  if (sizeCategory) {
+    return sizeCategory;
+  } else {
+    return AS_UIContentSizeCategoryUnspecified();
+  }
+}
```

Comment:
Nit, feel free to ignore this: It's a bit shorter to use ternary operator here, i.e:

 `return sizeCategory ? sizeCategory : AS_UIContentSizeCategoryUnspecified();`